src/core/render/index.js

```diff
@@ -52,13 +63,14 @@ function renderMain(html) {
   if (
     this.config.executeScript !== false &&
     typeof window.Vue !== 'undefined' &&
-    !executeScript()
+    // !executeScript()
+    !getScript()
   ) {
-    setTimeout(_ => {
+    // setTimeout(_ => {
       const vueVM = window.__EXECUTE_RESULT__
       vueVM && vueVM.$destroy && vueVM.$destroy()
       window.__EXECUTE_RESULT__ = new window.Vue().$mount('#main')
-    }, 0)
+    // }, 0)
```

Comment:
The changes with getScript/executeScript prevents script tags from executing twice. The `setTimeout` isn't needed, at least as far as I can tell everything works for me fine. 

Maybe it was a hack to defer script execution to later so that things in the DOM are ready? If that's the case, users can for example use `jQuery.ready()` like normal (or etc).

Reply:
I think it will effect the custom scrip option which are defined in the markdown right ?

Reply:
> custom scrip option which are defined in the markdown right

Can you expand on this?